codespell_lib/_codespell.py

```diff
@@ -655,24 +655,37 @@ def parse_options(
     return options, parser, used_cfg_files
 
 
-def parse_ignore_words_option(ignore_words_option: List[str]) -> Set[str]:
+def parse_ignore_words_option(
+    ignore_words_option: List[str],
+) -> Tuple[Set[str], Set[str]]:
     ignore_words: Set[str] = set()
+    ignore_words_cased: Set[str] = set()
     if ignore_words_option:
         for comma_separated_words in ignore_words_option:
-            ignore_words.update(
-                word.strip() for word in comma_separated_words.split(",")
-            )
-    return ignore_words
+            for word in comma_separated_words.split(","):
+                word = word.strip()
+                if word == word.lower():
+                    ignore_words.add(word)
+                else:
+                    ignore_words_cased.add(word)
+    return (ignore_words, ignore_words_cased)
```

Comment:
We now have two categories of words to ignore when reading text:
* `ignore_words`: words with lowercase characters only
* `ignore_words_cased`: words with at least an uppercase character

This will indeed help if we want to ignore `Manuel` but not `manuel`, which is a common requirement.